sdcm/snitch_configuration.py

```diff
@@ -44,47 +43,19 @@ def _get_dc_suffix(self) -> str:
         else:
             return ""
 
-    def apply(self) -> bool:
+    def apply(self) -> None:
         """
         Apply snitch configuration to the node (only in cassandra-rackdc.properties, scylla.yaml must be updated separately.).
 
         Returns `True` if require Scylla restart to make the effect.
         """
-        requires_restart = False
-        properties = {name: str(getattr(self, name)).lower() for name in dir(self)
-                      if isinstance(getattr(self.__class__, name, None), property)}
+        properties = {
+            'dc': self._datacenter,
+            'rack': self._rack,
+            'prefer_local': True,
```

Comment:
It will fail on K8S: https://github.com/scylladb/scylla-operator/issues/797

Reply:
it's executed only for multi-dc and simulated racks.